lib/coverartarchive-api.ts

```diff
@@ -37,6 +37,15 @@ export class CoverArtArchiveApi {
         Accept: "application/json"
       }
     });
+
+    const contentType = response.headers.get("Content-Type");
+    if (contentType?.toLowerCase() !== "application/json" && response.status === 404) {
```

Comment:
```suggestion
    if (response.status === 404 && contentType?.toLowerCase() !== "application/json") {
```
If you turn the conditions around, this combined expression is much faster.